h2o-core/src/main/java/water/rapids/ast/prims/string/algorithms/LevenstheinDistanceComparator.java

```diff
@@ -0,0 +1,79 @@
+package water.rapids.ast.prims.string.algorithms;
+
+import no.priv.garshol.duke.Comparator;
+
+import static org.apache.commons.lang.math.IEEE754rUtils.min;
+
+/**
```

Comment:
Did the original code have a license in the file?

Ideally we should follow this process: https://opensource.stackexchange.com/a/5161

Reply:
There is Apache 2 license https://github.com/Simmetrics/simmetrics/blob/master/simmetrics-core/src/main/java/org/simmetrics/metrics/Levenshtein.java

Reply:
Then please include this license in this file